### ISSUE TITLE
first open not work properly

### ISSUE THREAD

**Issue by ninomirabile** [arkon/ng-sidebar#67]
my component.ts
```typescript
import { Component, OnInit } from '@angular/core';

@Component({
  selector: 'app-sidebar',
  templateUrl: './sidebar.component.html',
  styleUrls: ['./sidebar.component.css']
})
export class SidebarComponent  {

  headerLogoImage : string;

  constructor() {
    this.headerLogoImage = './src/app/imgPath/fuel.svg'
   }

  private _opened: boolean = true;
  private _autoCollapseWidth: number = 500;
  private _autoCollapseHeight: number = 500;
  private _closeOnClickOutside: boolean = true;
  private _showBackdrop: boolean = true;  
  

  private _toggleSidebar() {
    this._opened = !this._opened;    
  }
}
```

my template

```html
<ng-sidebar-container>

      <!-- A sidebar -->
      <ng-sidebar
       [(opened)]="_opened"
       [mode]="'push'"
       [autoCollapseHeight]="_autoCollapseHeight"
       [autoCollapseWidth]="_autoCollapseWidth"
       [closeOnClickOutside]="_closeOnClickOutside"
       [showBackdrop]="_showBackdrop">
        <ul>
            <li>home</li>
            <li>prezzi</li>
            <li>carico</li>
            <li>contatori</li>
            <li>fatture</li>
            <li>impostazioni</li>
        </ul>
    
      </ng-sidebar>
</ng-sidebar-container>
```

when i run the first time the _opened works but it take all page size and backdrop not works. if i close and open -> sidebar works correctly.

**Comment by arkon** [comment]
Did you put it inside an `<ng-sidebar-container>`?

**Comment by ninomirabile** [comment]
my  complete template


![cattura](https://cloud.githubusercontent.com/assets/6699502/25027148/f32fd17a-20aa-11e7-8345-3a157541ab0b.PNG)

**Comment by arkon** [comment]
I just published v4.1.1, which should at least fix the backdrop not showing the first time. I'm not sure what the page size issue is though, that could be related to your specific content and styles.

**Comment by ninomirabile** [comment]
yes, i try sidebar in a simple test project, size is correct. thanks :)
